src/components/PersonalData/PersonalData.test.tsx

```diff
@@ -25,14 +24,23 @@ describe('Personal Data component', () => {
   let html: RenderResult
 
   it('renders the greeting without a name', () => {
-    renderWithAuthAndApollo(<PersonalData />, { user: null })
+    renderWithAuthAndApollo(
+      <PersonalData userDisplayName="BERNADETTE CAMPBELL" />,
+      {
+        user: null,
+      }
+    )
     const greeting = screen.getByRole('heading', { level: 2 })
-    expect(greeting).toHaveTextContent('Welcome!')
+    expect(greeting).toHaveTextContent('Welcome, BERNADETTE CAMPBELL')
   })
 
   describe('when logged in', () => {
     beforeEach(() => {
-      html = renderWithAuthAndApollo(<PersonalData />, {}, getDisplayNameMock)
+      html = renderWithAuthAndApollo(
+        <PersonalData userDisplayName="BERNADETTE CAMPBELL" />,
+        {},
+        getDisplayNameMock
```

Comment:
Question: Do we need to mock out the Apollo call here? It doesn't appear that `PersonalData` will be making any api calls now correct? Same with the test above on line 28.